app/models/ingredient.rb

```diff
@@ -1,3 +1,14 @@
 class Ingredient < ApplicationRecord
    validates_presence_of :name, :cost
+   has_many :recipe_ingredients
+   has_many :recipes, through: :recipe_ingredients
+
+   def self.total_cost
+      sum(:cost)
```

Comment:
Be careful with this implementation - the user story says we need to know the total cost of all ingredients _on a recipe_, so if thi sis being used on an association on Recipe, then it would be ok. But if this lets us know total cost of any collection of Ingredients, we need to be sure it's tested in both ways. 

An alternative would be to implement this as an instance method on the Recipe model: 
```ruby
def total_cost
   ingredients.sum(:cost)
end
```

Reply:
Either way, since you made new model methods you must also test them in `spec/models`. Models should consistently stay at 100% coverage.